.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
need to set nodeLinker to node-modules so that Yarn doesn't use the new default of Plug 'n' Play